docs/tutorial.dox

```diff
@@ -677,6 +682,11 @@ data file.
 - simple_xy_nc4_wr.c
 - simple_xy_nc4_rd.c
 
+\section filter_example The filter example
+This example demonstrates how to write and read a variable that is
+compressed using, in this example, bzip2 compression.
```

Comment:
Using bzip2 is a very nice new feature. The internet has this to say: "bzip2 has notably better compression ratio than gzip, which has to be the reason for the popularity of bzip2; it is slower than gzip especially in decompression and uses more memory."

So users will want to know about this new capability.